	* glib-convert.adb, glib-module.adb, gdk-pixbuf.adb, gtk-accel_map.adb,
	gtk-file_chooser.adb, gtk-file_selection.adb, gtk-icon_factory.adb,
	gtk-icon_theme.adb, gtk-image.adb, gtk-rc.adb, gtk-ui_manager.adb,
	gtk-window.adb: Fix import names for Windows.

Tue Jul  9 11:48:58 2002  Arnaud Charlet  <charlet@gnat.com>

	* glib-values.ads: Fix definition of GValue to conform to ms bitfields
	packing under Windows.

*** glib-values.ads	4 Sep 2002 14:47:59 -0000	1.13
--- glib-values.ads	8 Nov 2002 11:49:02 -0000
***************
*** 136,142 ****
     --  also provided inside Gtk.Text_Iter.
  
  private
!    type GValue_Data is array (1 .. 2) of Guint64;
     type GValue is record
        g_type : GType := GType_Invalid;
        data   : GValue_Data;
--- 136,143 ----
     --  also provided inside Gtk.Text_Iter.
  
  private
! 
!    type GValue_Data is array (1 .. 20) of Guint8;
     type GValue is record
        g_type : GType := GType_Invalid;
        data   : GValue_Data;
Index: glib-convert.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/glib-convert.adb,v
retrieving revision 1.11
diff -u -U0 -r1.11 glib-convert.adb
--- glib-convert.adb	16 Aug 2006 09:50:43 -0000	1.11
+++ glib-convert.adb	16 Jan 2007 14:47:50 -0000
@@ -203 +203 @@
-      pragma Import (C, Internal, "g_filename_from_uri");
+      pragma Import (C, Internal, "g_filename_from_uri_utf8");
@@ -227 +227 @@
-      pragma Import (C, Internal, "g_filename_from_utf8");
+      pragma Import (C, Internal, "g_filename_from_utf8_utf8");
@@ -251 +251 @@
-      pragma Import (C, Internal, "g_filename_to_uri");
+      pragma Import (C, Internal, "g_filename_to_uri_utf8");
@@ -285 +285 @@
-      pragma Import (C, Internal, "g_filename_to_utf8");
+      pragma Import (C, Internal, "g_filename_to_utf8_utf8");
Index: glib-module.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/glib-module.adb,v
retrieving revision 1.1
diff -u -U0 -r1.1 glib-module.adb
--- glib-module.adb	4 Dec 2000 10:04:11 -0000	1.1
+++ glib-module.adb	16 Jan 2007 14:47:50 -0000
@@ -57 +57 @@
-      pragma Import (C, Internal, "g_module_open");
+      pragma Import (C, Internal, "g_module_open_utf8");
@@ -116 +116 @@
-      pragma Import (C, Internal, "g_module_name");
+      pragma Import (C, Internal, "g_module_name_utf8");
Index: gdk-pixbuf.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gdk-pixbuf.adb,v
retrieving revision 1.7
diff -u -U0 -r1.7 gdk-pixbuf.adb
--- gdk-pixbuf.adb	15 Feb 2006 16:02:20 -0000	1.7
+++ gdk-pixbuf.adb	16 Jan 2007 14:47:50 -0000
@@ -105 +105 @@
-      pragma Import (C, Internal, "gdk_pixbuf_new_from_file");
+      pragma Import (C, Internal, "gdk_pixbuf_new_from_file_utf8");
@@ -126 +126 @@
-      pragma Import (C, Internal, "gdk_pixbuf_animation_new_from_file");
+      pragma Import (C, Internal, "gdk_pixbuf_animation_new_from_file_utf8");
Index: gtk-accel_map.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gtk-accel_map.adb,v
retrieving revision 1.9
diff -u -U0 -r1.9 gtk-accel_map.adb
--- gtk-accel_map.adb	21 Jul 2006 12:18:50 -0000	1.9
+++ gtk-accel_map.adb	16 Jan 2007 14:47:50 -0000
@@ -55 +55 @@
-      pragma Import (C, Internal, "gtk_accel_map_save");
+      pragma Import (C, Internal, "gtk_accel_map_save_utf8");
@@ -66 +66 @@
-      pragma Import (C, Internal, "gtk_accel_map_load");
+      pragma Import (C, Internal, "gtk_accel_map_load_utf8");
Index: gtk-file_chooser.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gtk-file_chooser.adb,v
retrieving revision 1.1
diff -u -U0 -r1.1 gtk-file_chooser.adb
--- gtk-file_chooser.adb	5 Oct 2006 15:28:44 -0000	1.1
+++ gtk-file_chooser.adb	16 Jan 2007 14:47:50 -0000
@@ -63 +63 @@
-      pragma Import (C, Internal, "gtk_file_chooser_add_shortcut_folder");
+      pragma Import (C, Internal, "gtk_file_chooser_add_shortcut_folder_utf8");
@@ -117 +117 @@
-      pragma Import (C, Internal, "gtk_file_chooser_get_filename");
+      pragma Import (C, Internal, "gtk_file_chooser_get_filename_utf8");
@@ -141 +141 @@
-      pragma Import (C, Internal, "gtk_file_chooser_get_filenames");
+      pragma Import (C, Internal, "gtk_file_chooser_get_filenames_utf8");
@@ -154 +154 @@
-      pragma Import (C, Internal, "gtk_file_chooser_get_current_folder");
+      pragma Import (C, Internal, "gtk_file_chooser_get_current_folder_utf8");
@@ -241 +241,2 @@
-      pragma Import (C, Internal, "gtk_file_chooser_get_preview_filename");
+      pragma Import
+        (C, Internal, "gtk_file_chooser_get_preview_filename_utf8");
@@ -404 +405,2 @@
-      pragma Import (C, Internal, "gtk_file_chooser_list_shortcut_folders");
+      pragma Import
+        (C, Internal, "gtk_file_chooser_list_shortcut_folders_utf8");
@@ -457 +459,2 @@
-      pragma Import (C, Internal, "gtk_file_chooser_remove_shortcut_folder");
+      pragma Import
+        (C, Internal, "gtk_file_chooser_remove_shortcut_folder_utf8");
@@ -503 +506 @@
-      pragma Import (C, Internal, "gtk_file_chooser_select_filename");
+      pragma Import (C, Internal, "gtk_file_chooser_select_filename_utf8");
@@ -531 +534 @@
-      pragma Import (C, Internal, "gtk_file_chooser_set_current_folder");
+      pragma Import (C, Internal, "gtk_file_chooser_set_current_folder_utf8");
@@ -603 +606 @@
-      pragma Import (C, Internal, "gtk_file_chooser_set_filename");
+      pragma Import (C, Internal, "gtk_file_chooser_set_filename_utf8");
@@ -729 +732 @@
-      pragma Import (C, Internal, "gtk_file_chooser_unselect_filename");
+      pragma Import (C, Internal, "gtk_file_chooser_unselect_filename_utf8");
Index: gtk-file_selection.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gtk-file_selection.adb,v
retrieving revision 1.27
diff -u -U0 -r1.27 gtk-file_selection.adb
--- gtk-file_selection.adb	29 May 2006 16:39:12 -0000	1.27
+++ gtk-file_selection.adb	16 Jan 2007 14:47:50 -0000
@@ -151 +151 @@
-      pragma Import (C, Internal, "gtk_file_selection_get_filename");
+      pragma Import (C, Internal, "gtk_file_selection_get_filename_utf8");
@@ -294 +294 @@
-      pragma Import (C, Internal, "gtk_file_selection_set_filename");
+      pragma Import (C, Internal, "gtk_file_selection_set_filename_utf8");
@@ -353 +353 @@
-      pragma Import (C, Internal, "gtk_file_selection_get_selections");
+      pragma Import (C, Internal, "gtk_file_selection_get_selections_utf8");
Index: gtk-icon_factory.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gtk-icon_factory.adb,v
retrieving revision 1.8
diff -u -U0 -r1.8 gtk-icon_factory.adb
--- gtk-icon_factory.adb	12 Oct 2006 12:10:09 -0000	1.8
+++ gtk-icon_factory.adb	16 Jan 2007 14:47:50 -0000
@@ -201 +201 @@
-      pragma Import (C, Internal, "gtk_icon_source_set_filename");
+      pragma Import (C, Internal, "gtk_icon_source_set_filename_utf8");
@@ -522 +522 @@
-      pragma Import (C, Internal, "gtk_icon_source_get_filename");
+      pragma Import (C, Internal, "gtk_icon_source_get_filename_utf8");
Index: gtk-icon_theme.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gtk-icon_theme.adb,v
retrieving revision 1.1
diff -u -U0 -r1.1 gtk-icon_theme.adb
--- gtk-icon_theme.adb	12 Oct 2006 12:05:34 -0000	1.1
+++ gtk-icon_theme.adb	16 Jan 2007 14:47:50 -0000
@@ -115 +115 @@
-      pragma Import (C, Internal, "gtk_icon_info_get_filename");
+      pragma Import (C, Internal, "gtk_icon_info_get_filename_utf8");
@@ -166 +166 @@
-      pragma Import (C, Internal, "gtk_icon_theme_append_search_path");
+      pragma Import (C, Internal, "gtk_icon_theme_append_search_path_utf8");
@@ -236 +236 @@
-      pragma Import (C, Internal, "gtk_icon_theme_get_search_path");
+      pragma Import (C, Internal, "gtk_icon_theme_get_search_path_utf8");
@@ -372 +372 @@
-      pragma Import (C, Internal, "gtk_icon_theme_prepend_search_path");
+      pragma Import (C, Internal, "gtk_icon_theme_prepend_search_path_utf8");
@@ -421 +421 @@
-      pragma Import (C, Internal, "gtk_icon_theme_set_search_path");
+      pragma Import (C, Internal, "gtk_icon_theme_set_search_path_utf8");
Index: gtk-image.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gtk-image.adb,v
retrieving revision 1.22
diff -u -U0 -r1.22 gtk-image.adb
--- gtk-image.adb	22 May 2006 15:50:42 -0000	1.22
+++ gtk-image.adb	16 Jan 2007 14:47:50 -0000
@@ -242 +242 @@
-      pragma Import (C, Internal, "gtk_image_new_from_file");
+      pragma Import (C, Internal, "gtk_image_new_from_file_utf8");
@@ -336 +336 @@
-      pragma Import (C, Internal, "gtk_image_set_from_file");
+      pragma Import (C, Internal, "gtk_image_set_from_file_utf8");
Index: gtk-rc.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gtk-rc.adb,v
retrieving revision 1.20
diff -u -U0 -r1.20 gtk-rc.adb
--- gtk-rc.adb	12 Oct 2006 12:10:09 -0000	1.20
+++ gtk-rc.adb	16 Jan 2007 14:47:50 -0000
@@ -80 +80 @@
-      pragma Import (C, Internal, "gtk_rc_add_default_file");
+      pragma Import (C, Internal, "gtk_rc_add_default_file_utf8");
@@ -92 +92 @@
-      pragma Import (C, Internal, "gtk_rc_set_default_files");
+      pragma Import (C, Internal, "gtk_rc_set_default_files_utf8");
@@ -134 +134 @@
-      pragma Import (C, Internal, "gtk_rc_parse");
+      pragma Import (C, Internal, "gtk_rc_parse_utf8");
Index: gtk-ui_manager.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gtk-ui_manager.adb,v
retrieving revision 1.3
diff -u -U0 -r1.3 gtk-ui_manager.adb
--- gtk-ui_manager.adb	12 Oct 2006 12:11:11 -0000	1.3
+++ gtk-ui_manager.adb	16 Jan 2007 14:47:50 -0000
@@ -82 +82 @@
-      pragma Import (C, Internal, "gtk_ui_manager_add_ui_from_file");
+      pragma Import (C, Internal, "gtk_ui_manager_add_ui_from_file_utf8");
Index: gtk-window.adb
===================================================================
RCS file: /paris.a/cvs/Dev/GtkAda/src/gtk-window.adb,v
retrieving revision 1.47
diff -u -U0 -r1.47 gtk-window.adb
--- gtk-window.adb	17 May 2006 16:17:44 -0000	1.47
+++ gtk-window.adb	16 Jan 2007 14:47:50 -0000
@@ -731 +731 @@
-      pragma Import (C, Internal, "gtk_window_set_icon_from_file");
+      pragma Import (C, Internal, "gtk_window_set_icon_from_file_utf8");
@@ -889 +889,2 @@
-      pragma Import (C, Internal, "gtk_window_set_default_icon_from_file");
+      pragma Import
+        (C, Internal, "gtk_window_set_default_icon_from_file_utf8");
